packages/cog/src/cog/cog.ts

```diff
@@ -22,11 +22,11 @@ export function onProgress(keys: Record<string, any>, logger: LogType): (p: numb
 /**
  * Return the width/height of the quadkey in pixels at the target resolution
  * @param quadKey
- * @param targetResolution
+ * @param targetResolutionZoom
```

Comment:
What is a resolution zoom?

is this pixels / meters? or a `zoom`?

Reply:
what about just `targetZoom`?